### ISSUE TITLE
[BUG]: Affine version in requirements.txt doesn't exist

### ISSUE THREAD

**Issue by dannyjeck-matroid** [CosmiQ/solaris#263]
_Thank you for helping us improve `solaris`!_

## Summary of the bug
In requirements.txt affine==2.2.3 doesn't exist

## Steps to reproduce the bug

```
pip install -r requirements.txt
```

## Buggy behavior and/or error message
Please describe the buggy behavior and/or paste output here.
```
Collecting affine==2.2.3 (from -r requirements.txt (line 18))
  ERROR: Could not find a version that satisfies the requirement affine==2.2.3 (from -r requirements.txt (line 18)) (from versions: 1.0, 1.0.1, 1.1.0, 1.2.0, 1.3.0, 2.0b1, 2.0b2, 2.0.0.post1, 2.1.0, 2.2.0, 2.2.1, 2.2.2, 2.3.0)
ERROR: No matching distribution found for affine==2.2.3 (from -r requirements.txt (line 18))
```

## Expected behavior
I expected to be able to install the dependencies

## Environment information
- OS: Ubuntu
- `solaris` version: master (cf372fe5302e62a4b96dbbf291dbbc228b9aa866)
- python version: 3.7
- version of any relevant dependencies (optional - we may ask for this information later if not provided)

## Additional context
I was doing this inside of a docker file, but looking online it appears as though no 2.2.3 version of the affine library exists on pip.

**Comment by nrweir** [comment]
@dannyjeck-matroid thanks for the catch. I'll fix this today.

**Comment by nrweir** [comment]
Resolved with #264. Bumped the affine version to 2.3.0.
